apps/src/music/views/patternPanel.module.scss

```diff
@@ -13,10 +13,15 @@
   }
 }
 
-.name {
+.nameContainer {
   display: inline-block;
   width: 80px;
   vertical-align: middle;
+  padding-left: 3px;
+
+  .name {
```

Comment:
Maybe as a follow-up - should we have some sort of opacity change (and maybe even a translucent 'play' icon that appears over the label) on hover?

Reply:
Good question.  Agreed that this doesn't feel like final UI, but I wasn't quite sure what else we should do.  I think once we play with this for a bit it might become more clear!